src/client/Microsoft.Identity.Client/Http/LinearRetryStrategy.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Identity.Client.Http
+{
+    internal class LinearRetryStrategy
```

Comment:
An object with no state is a code smell. It takes up space on the stack. See my previous comment

Reply:
resolved offline during feature presentation to team